lib/commands/nw-test/index.js

```diff
@@ -106,9 +213,15 @@ module.exports = {
     var _this = this;
     options.outputPath = this.tmp();
 
-    var buildTask = new this.tasks.Build(this.taskOptions());
-
-    return buildTask.run({
+    var promise;
+    // server mode will build continuously
+    if (options.server) {
+      options.environment = 'development';
```

Comment:
My original thought was that the usually when you decide to use the `--server` flag, you're developing code not in a CI environment. However I think I actually added it to see if I could some how capture that environment in the built ember cli's application environment that goes into `index.html`. But I think we should just leave this out and let the user decide what environment to use.